upcoming_changes/177.enhancements.rst

```diff
@@ -0,0 +1,3 @@
+- `s.crop_edges` moved to utils. It also now can take 1,2 or 4 values
+as input (can crop each axis independently) and also accepts percentages.
+The use of it under the `CommonLumiSpectrum` class is deprecated.
```

Comment:
```suggestion
`crop_edges` moved to utils to allow consistent cropping for a list of signals. It also now
can take 1,2 or 4 values as input (can crop each axis independently) and accepts HyperSpy fancy indexing.
It can still be used for a single signal through the :class:`CommonLumi` class, but using
the parameter ``crop_px`` will be deprecated in the future.
```

Reply:
See https://lumispy--237.org.readthedocs.build/en/237/changelog.html for the formatted version (the bullet point is automatically added)